user.rb

```diff
@@ -0,0 +1,21 @@
+class User
+  attr :hoge
+  def initialize
+    @hoge =    [a,b,c,d]*''
+    # にゃーん
+  end
+
+  def get_hoge
+    @hoge
+  end
+
+  def poyo
+    1 + 1
+  end
+
+  def foo
+    hoge {
```

Comment:
__[staging][rubocop]__
* Avoid using `{...}` for multi-line blocks. [:ref](https://github.com/bbatsov/ruby-style-guide#single-line-blocks)

<!-- Comment by [SideCI](https://sideci.com) -->